src/renderers/dom/client/syntheticEvents/SyntheticEvent.js

```diff
@@ -135,9 +135,15 @@ Object.assign(SyntheticEvent.prototype, {
 
     if (event.stopPropagation) {
       event.stopPropagation();
-    } else {
+    } else if (typeof event.cancelBubble !== 'unknown') { // eslint-disable-line valid-typeof
```

Comment:
I'm care-face on this (not my decision). If it's a problem just leave it as is and they'll complain if need be.